merlin-server/sql/merlin/tables/constraint_run.sql

```diff
@@ -0,0 +1,49 @@
+create type constraint_status as enum('resolved', 'constraint-outdated', 'simulation-outdated');
+
+create table constraint_run (
+  constraint_id integer not null,
+  constraint_definition text not null,
+  plan_id integer not null,
+  simulation_dataset_id integer not null,
+
+  status constraint_status not null default 'resolved',
+  violations jsonb null,
+
+  -- Additional Metadata
+  requested_by text,
+  requested_at timestamptz not null default now(),
+
+  constraint constraint_run_to_constraint
+    foreign key (constraint_id)
+      references "constraint"
+      on delete cascade,
+  constraint constraint_run_to_simulation_dataset
+    foreign key (simulation_dataset_id)
+      references simulation_dataset
+      on delete cascade,
+  constraint constraint_run_requested_by
+    foreign key (requested_by)
+      references metadata.users
+      on update cascade
+      on delete set null
+);
```

Comment:
This table needs a Primary Key

Reply:
Additionally, we may want to add an `index` on this table on `simulation_dataset_id`.